app/helpers/tooltips_helper.rb

```diff
@@ -24,6 +24,14 @@ def your_call_list_help_text
     text.strip
   end
 
+  def budget_bar_help_text
```

Comment:
Can we get a super simple test for this that asserts a match with the words `budget bar`, just so if we ever have to revisit this we don't accidentally muck it up? something like:

```
# in test/helpers/tooltips_helper_test.rb
assert_match /budget bar/, budget_bar_help_text
```